### PR TITLE
[service.subtitles.opensubtitles-com] 1.0.0

### PR DESCRIPTION
**Pull request by opensubtitlesdev**
### Description
This extension is the new official opensubtitles add-on, using the new REST API from opensubtitles.com.
The code is based on the previous extension, but will now evolve on its own, maintained by the main opensubtitles developers. Users are still able to use the previous extension, but are encouraged to upgrade.

Opensubtitles.com is the new opensubtitles website,  who will in term completely replace opensubtitles.org. It uses a totally new database, old accounts can be imported [here](https://www.opensubtitles.com/en/users/import) but will not work with this extension.

Register for a new account [here](https://www.opensubtitles.com/en/users/sign_up) if you are new to opensubtitles

### Checklist:

-  [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
-  [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
-  [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

### CODE REVIEW

**Comment by romanvm** [comment]
Please remove compiled .mo language files. Kodi does not use gettext proper so those files are not needed.

**Comment by romanvm** [comment]
Also please remove compiled python bytecode files along with `__pycache__` directories. Such files and directories (and .mo files as well) should not be commited to Git.

**Comment by romanvm** [comment]
Please remove empty files as well.

**Review by romanvm — CHANGES_REQUESTED** [review]

File: service.subtitles.opensubtitles-com/resources/lib/cache.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
```
[the remark this answers is not in the document]
Comment (opensubtitlesdev) [reply service.subtitles.opensubtitles-com/resources/lib/cache.py:1]: ok, I removed the empty files, encoding comments, and compiled mo as suggested.

File: service.subtitles.opensubtitles-com/resources/lib/data_collector.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    return xbmc.Player().getPlayingFile()
+
+
+def get_media_data():
+    item = {"year": xbmc.getInfoLabel("VideoPlayer.Year"),
+            "season_number": str(xbmc.getInfoLabel("VideoPlayer.Season")),
+            "episode_number": str(xbmc.getInfoLabel("VideoPlayer.Episode")),
+            "tv_show_title": normalize_string(xbmc.getInfoLabel("VideoPlayer.TVshowtitle")),
+            "original_title": normalize_string(xbmc.getInfoLabel("VideoPlayer.OriginalTitle")),
+            "imdb_id": xbmc.getInfoLabel("VideoPlayer.IMDBNumber")}
```
[the remark this answers is not in the document]
Comment (opensubtitlesdev) [reply service.subtitles.opensubtitles-com/resources/lib/data_collector.py:24]: Our API works better if querying sending the file name, and moviehash, so I removed all imdb id's and unique ID's reference.

**Comment by romanvm** [comment]
Do you really need empty `.placeholder` file?

**Comment by jujudellago** [comment]
probably not.... no idea why it was there, I removed it.

**Review by romanvm — APPROVED** [review]

**Pull request merged**